comparitors.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+var bigNum = function(value1,value2){
+    if(value1>value2){
+        console.log("Value 1 is bigger!");
+    }
+    else{
+        console.log("Value 2 is bigger");
+    }
+}
+bigNum(2,1);
+bigNum(3,4);
+
+
+var posProd = function(value1,value2,value3){
+    var result = (value1*value2)*value3;
+    console.log("The product is" + result);
+    if(result>0){
+        console.log("Product is positive!");
+    }
+    else{
+        console.log("Product is negative!");
+    }
+
+}
+
+posProd(1,2,3);
+posProd(1,2,-3);
+
+var gradeCalc=function(score){
+    if(score<40){
+        console.log("Your grade = F");
```

Comment:
I'd be tempted to just return the grade letter - the function is more re-usable that way.